numpy/random/_examples/cython/setup.py

```diff
@@ -11,18 +11,17 @@
 from setuptools.extension import Extension
 from os.path import join, abspath, dirname
 
-curpath = abspath(dirname(__file__))
+path = abspath(dirname(__file__))
```

Comment:
Why this change? `path` is, I think, too generic.

Reply:
7 letters made the line spill below. Excessive beautification.